content/en/lotus/manage/chain-management.md

```diff
@@ -65,15 +52,6 @@ These lightweight state snapshots **do not contain any message receipts**. To ge
     lotus daemon --import-snapshot minimal_finality_stateroots_517061_2021-02-20_11-00-00.car
     ```
 
-{{< alert icon="tip" >}}
-We strongly recommend that you download and verify the checksum of the snapshot before importing. However, you can skip the `sha256sum` check and use the snapshot URL directly if you prefer:
-
-```shell
-lotus daemon --import-snapshot https://snapshots.mainnet.filops.net/minimal/latest
-```
-
-{{< /alert >}}
-
 #### New Lightweight Snapshot Service
```

Comment:
@Reiers does this paragraph need to exist anymore?

Reply:
No hence the "-", I removed it.